tests/integration/block_sdk_suite.go

```diff
@@ -1258,3 +1266,161 @@ func (s *IntegrationTestSuite) TestLanes() {
 		require.Equal(s.T(), user2BalanceBefore, user2BalanceAfter+delegation.Amount.Int64())
 	})
 }
+
+func (s *IntegrationTestSuite) TestNetwork() {
+	amountToTest := time.NewTicker(time.Second * 45)
+	defer amountToTest.Stop()
+
+	numTxs := 10
+	sendAmount := sdk.NewCoins(sdk.NewCoin(s.denom, math.NewInt(100)))
+	delegation := sdk.NewCoin(s.denom, math.NewInt(100))
+	validators := QueryValidators(s.T(), s.chain.(*cosmos.CosmosChain))
+
+	s.Run("can produce blocks with only default transactions", func() {
+		for {
+			select {
+			case <-amountToTest.C:
+				return
+			default:
+				height, err := s.chain.(*cosmos.CosmosChain).Height(context.Background())
+				require.NoError(s.T(), err)
```

Comment:
```suggestion
				s.Require().NoError(err)
```

here and other places in the tests